src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1182,6 +1161,46 @@ components:
         message_hex:
           $ref: "#/components/schemas/HexString"
 
+    #
+    # /stream/* Request & Response Types
+    #
+
+    StreamTransactionsRequest:
```

Comment:
I know that sbor encoding is not possible to address now but what with other mixins:

- raw_payload
- Parts of the receipt (state_changes, fee_summary, events)

is it something we're skipping or delaying?

Reply:
This is definitely outside the scope of this PR and I'm personally not even sure how handle that.